### DISCUSSION TITLE
firefly-iii exposed to the web

### DISCUSSION THREAD

**Discussion by gianmarcoodorizzi** [firefly-iii/firefly-iii#8062] [General]
I've recently bought a shared Linux hosting and installed Firefly III on it (also by importing the database from a local installation).
The web root is:
_/web/htdocs/www.mydomain.com/home_

and firefly-iii folder is in that directory, thus exposing its content to the web.
Unfortunately, I don't have write access to www.mydomain.com folder, so I cannot place firefly-iii there.
Is there a decent workaround for this?
I am also struggling to set up mod_rewrite as I keep getting 404s, but that's a different issue for now.

Thank you in advance!

**Comment by JC5** [comment]
Not really no. It will automatically redirect and it should hide sensitive content.

**Comment by gianmarcoodorizzi** [comment]
Thank you!
